lang/java/compiler/src/main/velocity/org/apache/avro/compiler/specific/templates/java/classic/record.vm

```diff
@@ -338,7 +338,7 @@ static {
      */
     private Builder(#if ($schema.getNamespace())$this.mangle($schema.getNamespace()).#end${this.mangle($schema.getName())} other) {
 #if ($schema.isError())      super(other)#else
-      super(SCHEMA$)#end;
+      super(SCHEMA$,MODEL$)#end;
```

Comment:
Thanks @RyanSkraba for the review
```suggestion
      super(SCHEMA$, MODEL$)#end;
```
@unchuckable Don't forget this one :)

Reply:
Dang. Hereby I propose a feature: run google formatter over generated code 😉